src/options.ts

```diff
@@ -81,6 +81,19 @@ export const getSourcemaps = (): string[] => {
   return sourcemapsOption.split(' ');
 };
 
+/**
+ * Dist is optional, but should be a string when provided.
+ * @returns string
+ */
+export const getDist = (): string | undefined => {
+  const distOption: string = core.getInput('dist');
+  if (!distOption) {
+    return null;
```

Comment:
I'm seeing this error in the CI logs.
> TS2322: Type 'null' is not assignable to type 'string | undefined'.

Would returning `undefined` work?

Reply:
i think this was my mistake, i originally has this function configured to return undefined, but changed it to `null` after seeing all the other properties use `null` not `undefined`... I just didn't update the return type.

vscode corrected me as soon as i opened the PR, not sure why it didn't point it out earlier.

hopefully this is enough for a CI pass 🤞

Reply:
can see some of the `sentry-cli` integration tests use `null` so I expect this is correct.

<img width="663" alt="Screen Shot 2023-04-03 at 12 00 26 PM" src="https://user-images.githubusercontent.com/153077/229386252-e69d7ed6-df97-443b-97d9-eecd8d8a56d6.png">

Reply:
made this change and committed 👍